lib/av_client/crypto/vote_encoder.ts

```diff
@@ -0,0 +1,126 @@
+import * as sjcl from '../sjcl';
+import * as crypto from "../aion_crypto";
+
+const ADJUSTING_BYTE_COUNT = 1
+
+/**
+ * @param {bytes} the bytes to be encoded as points.
+ * @return {array of sjcl.ecc.point} An array of points representing encoding the bytes
+ */
+export const bytesToPoints = (bytes: Uint8Array): any[] => {
+    const pointContentSize = computePointContentSize()
+    const pointCount = Math.ceil(bytes.length / pointContentSize)
+
+    // pad the byte array with 0x00 on the right side
+    const paddedBytes = padBytes(pointCount * pointContentSize, bytes)
```

Comment:
could be called padBytesRight with bytes as first argument

Reply:
done.